docs/source/markdown/podman-create.1.md

```diff
@@ -408,6 +408,38 @@ The number of retries allowed before a healthcheck is considered to be unhealthy
 The initialization time needed for a container to bootstrap. The value can be expressed in time format like
 `2m3s`. The default value is `0s`
 
+#### **--health-startup-cmd**=*"command"* | *'["command", "arg1", ...]'*
+
+Set a startup healthcheck command for a container. This command will be executed inside the container and is used to gate the regular healthcheck.
+When the startup command succeeds, the regular healthcheck will begin and the startup healthcheck will cease.
+Optionally, if the command fails for a set number of attempts, the container will be restarted.
+A startup healthcheck can be used to ensure that containers with an extended startup period are not marked as unhealthy until they are fully started.
+Startup healthchecks can only be used when a regular healthcheck (from the container's image or the **--health-cmd** option) is also set.
+
+#### **--health-startup-interval**=*interval*
+
+Set an interval for the startup healthcheck.
+An _interval* or **disable** results in no automatic timer setup.
```

Comment:
```suggestion
An _interval* or the value **disable** results in no automatic timer setup.
```
I'm not sure what the right md syntax is for the word "interval" here is, but I don't think it's right now.